cmd/oras/internal/option/packer.go

```diff
@@ -69,6 +72,25 @@ func (opts *Packer) ExportManifest(ctx context.Context, fetcher content.Fetcher,
 	}
 	return os.WriteFile(opts.ManifestExportPath, manifestBytes, 0666)
 }
+func (opts *Packer) Parse() error {
+	var failedPaths []string
+	if !opts.PathValidationDisabled {
```

Comment:
nit: put `failedPaths` in the proper scope.
```suggestion
	if !opts.PathValidationDisabled {
		var failedPaths []string
```